src/Services/CareTeamService.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * CareTeamService
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Yash Bothra <yashrajbothra786gmail.com>
+ * @copyright Copyright (c) 2020 Yash Bothra <yashrajbothra786gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace OpenEMR\Services;
+
+use OpenEMR\Common\Uuid\UuidRegistry;
+use OpenEMR\Common\Uuid\UuidMapping;
+use OpenEMR\Validators\BaseValidator;
+use OpenEMR\Validators\ProcessingResult;
+
+class CareTeamService extends BaseService
+{
+
+    private const PATIENT_TABLE = "patient_data";
+    private const PRACTITIONER_TABLE = "users";
+    private const FACILITY_TABLE = "facility";
+    private $uuidRegistry;
+
+    /**
+     * Default constructor.
+     */
+    public function __construct()
+    {
+        $this->uuidMapping = new UuidMapping();
+        (new UuidRegistry(['table_name' => self::PATIENT_TABLE]))->createMissingUuids();
+        (new UuidRegistry(['table_name' => self::PRACTITIONER_TABLE]))->createMissingUuids();
+        (new UuidRegistry(['table_name' => self::FACILITY_TABLE]))->createMissingUuids();
+        $this->uuidMapping->createMissingResourceUuids("CareTeam", self::PATIENT_TABLE);
```

Comment:
will need the uuidMapping fix that discussed on call for this to work

Reply:
Yup Will push it asap :)